tls/s2n_server_extensions.c

```diff
@@ -191,3 +203,14 @@ int s2n_recv_server_sct_list(struct s2n_connection *conn, struct s2n_stuffer *ex
 
     return 0;
 }
+
+int s2n_recv_server_max_frag_len(struct s2n_connection *conn, struct s2n_stuffer *extension)
+{
+    uint16_t max_fragment_length;
+    GUARD(s2n_stuffer_read_uint16(extension, &max_fragment_length));
```

Comment:
This is because I am passing the actual mfl value not the mfl enum code. 
I chose to use uint16_t as max of uint8_t is 255 and all max fragment length values are over it. 

im not sure RFC6066's `enum MaxFragmentLength ` requires specifically uint8_t.

by on the wire, did you mean `s2n_server_extensions_send ` (during server hello send, writes mfl value with uint16_t size) or `s2n_client_extensions_send` (during client hello send, writes mfl code with uint8_t size) ?